lightning/src/ln/outbound_payment.rs

```diff
@@ -2254,6 +2280,14 @@ impl_writeable_tlv_based_enum_upgradable!(PendingOutboundPayment,
 		(4, retry_strategy, required),
 		(6, route_params, required),
 	},
+	// Added in 0.1. Prior versions will drop these outbounds on downgrade, which is safe because
+	// no HTLCs are in-flight.
+	(11, AwaitingOffer) => {
+		(0, expiration, required),
+		(2, retry_strategy, required),
+		(4, max_total_routing_fee_msat, option),
+		(6, amount_msats, required),
+	},
```

Comment:
just trying to learn here. I noted this in other places too. There is a specific reason that we use the number with a step of 2? is it probably for upgrade and downgrade?